README.md

```diff
@@ -51,4 +60,4 @@ protoc --doc_out=markdown,README.md:. *.proto
 Dropsonde is intended to be a "fire and forget" protocol, in the sense that an emitter should send events to its receiver with no expectation of acknowledgement. There is no "handshake" step; the emitter simply begins emitting to a known address of an expected recipient. 
 
 ### Heartbeat requests
-One of the control messages available is the [HeartbeatRequest](control/README.md#control.HeartbeatRequest). When a process implementing Dropsonde receives this message, it MUST respond with with a [Heartbeat](events/README.md#events.Heartbeat) message. That `Heartbeat` should include the same `UUID` as was received in the containing `ControlMessage`.
```

Comment:
I can't tell what the change is from the github UI, and I don't know what the "standard" is. I'm removing the trailing newline, hoping that was what you meant.